js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -106,65 +128,117 @@ export const ResourceRoot: React.FC<Props> = (props) => {
+                                      {' '}
+                                      <Link
+                                        to={workspacePathFromAddress(
+                                          repoAddress,
+                                          `/resources/${nestedResources[key].name}`,
+                                        )}
+                                      >
+                                        {resourceDisplayName(nestedResources[key])}
+                                      </Link>
+                                    </Tag>
+                                  </td>
+                                </tr>
+                              );
+                            })}
+                          </tbody>
+                        </Table>
+                      </Box>
+                    )}
+                    <Box>
+                      <SectionHeader>
+                        <Subheading>Configuration</Subheading>
+                      </SectionHeader>
+                      <Table>
+                        <thead>
+                          <tr>
+                            <th style={{width: 120}}>Key</th>
+                            <th style={{width: 90}}>Type</th>
+                            <th style={{width: 90}}>Value</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {resourceDetailsOrError.configFields.map((field) => {
+                            const defaultValue = field.defaultValueAsJson;
+                            const type =
+                              field.name in configuredValues
+                                ? configuredValues[field.name].type
+                                : null;
+                            const actualValue =
+                              field.name in configuredValues
+                                ? configuredValues[field.name].value
+                                : defaultValue;
+
+                            const isDefault = type === 'VALUE' && defaultValue === actualValue;
+                            return (
+                              <tr key={field.name}>
+                                <td>
+                                  <Box
+                                    flex={{direction: 'column', gap: 4, alignItems: 'flex-start'}}
+                                  >
+                                    <strong>{field.name}</strong>
+                                    <div style={{fontSize: 12, color: Colors.Gray700}}>
+                                      {field.description}
+                                    </div>
+                                  </Box>
+                                </td>
+                                <td>{remapName(field.configTypeKey)}</td>
+                                <td>
+                                  <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
```

Comment:
`space-between` could end up a little strange here if there is a lot of space for the cell. Maybe just a `gap` value?